features/search/works_info.feature

```diff
@@ -56,6 +56,30 @@ Feature: Search works by work info
       And I should see "fifth"
       And I should see "I am <strong>er Than Yesterday & Other Lies"
 
+  @new-search
+  Scenario: Search by status
+    Given I have loaded the fixtures
+    When I am on the search works page
+      And I choose "Only complete works"
+      And I press "Search" within "#new_work_search"
+    Then I should see "You searched for: Complete"
```

Comment:
The new tests look good!

Can you add a few checks that certain works show up for each new search (like `And I should see "3 Found"` from the other tests)? It's fine if you don't have time though, since it's not strictly related to the current issue.

Reply:
Sounds good. Is there a preference on using one of the work sets (Battle Set, etc) vs plain ol' fixtures? Seems to work for another test later down (maybe the one you're referencing):
```
  Scenario: Search and sort by title
    Given I have loaded the fixtures
...
      And I should see "3 Found"
```

Reply:
"Battle Set" has more steps to set up a challenge and takes longer so let's go with "loaded the fixtures" if possible.

Reply:
You don't need to, I think this test is fine.